README.rst

```diff
@@ -202,6 +202,18 @@ Screenshots
    :target: https://cloud.githubusercontent.com/assets/122800/11951695/f371b926-a88a-11e5-91c2-499166776bd3.png
    :alt: Enhanced HTML report
 
+Contributing
+------------
+
+Fork the repository and submit PRs with bug fixes and enhancements,  contributions are very welcome.
+
+Tests can be run locally with `tox`_, for example to execute tests for Python 2.7 and 3.6 execute::
+
+    tox -e py27,py36
```

Comment:
> The tox environment names include the two latest major stable pytest versions (needing an update since 3.1 was released).

I noticed this, but also noticed that `-e py27,py35` worked and used the latest pytest version, so just went with it because it seemed enough for doing a local test before submitting a PR. But if you prefer we can bless a test environment for local testing, say `-e py27-pytest31,py36-pytest31`.

FWIW we usually recommend `-e py27,py35,linting` on pytest.

>  it appears that py36 does work.. not sure if this is a bug or a feature.

To be honest I only tested `py35` and it worked, and as an afterthought I changed to `py36`. When you say it does not work, do you mean It does not run at all or it fails some tests?